type_templates/vrm.js

```diff
@@ -46,30 +46,37 @@ export default e => {
     // globalThis.app = app;
     // globalThis.avatarRenderer = avatarRenderer;
 
+    avatarRenderer.initControlObject(app);
+
     const _addPhysics = () => {
-      const fakeHeight = 1.5;
+      const HEAD_HEIGHT = 0.13; // head height is zero in initialization so we need to take it into account

```

Comment:
This seems arbitrary. We should try to get it from the skeleton.

Additionally, this looks like the same math as https://github.com/upstreet-labs/app/pull/48/files#diff-3b54efea8f0c97b79b516f87d0fd69b985ae4fabc07694791633c647b0d5bcbaR52 so should probably be abstracted.